src/test/java/com/epam/tc/hw2/ex2/Ex2.java

```diff
@@ -0,0 +1,168 @@
+package com.epam.tc.hw2.ex2;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+
+public class Ex2 {
+
+
+    WebDriver driver;
+
+    @BeforeSuite
+    static void setupAll() {
+        WebDriverManager.chromedriver().setup();
+
+    }
+
+    @BeforeTest
+    void setup() {
+        driver = new ChromeDriver();
+    }
```

Comment:
Code duplication. You use driver setup twice.

Reply:
Merged into one method. WebDriverManager should be used for easy setup without configuration. Without driver = new ChromeDriver(); WebDriverManager wont work.

Reply:
Nope, I meant that you have this code in 2 classes. The idea was to move this into parent class

Reply:
Ups, sure. I thought this homework required only 2 classes.

Reply:
Fixed